core/ir/x86/decode_table.c

```diff
@@ -10703,4 +10703,1547 @@ const instr_info_t extra_operands[] =
     {OP_CONTD, 0xc00f0171, "<enclv cont'd>", ecx, edx, edx, xx, xx, mrm, x, END_LIST},
 };
 
+const uint category_by_opcode[] = {
+    /* OP_INVALID */           DR_INSTR_CATEGORY_OTHER,
+    /* OP_UNDECODED */         DR_INSTR_CATEGORY_OTHER,
+    /* OP_CONTD */             DR_INSTR_CATEGORY_OTHER,
+    /* OP_LABEL */             DR_INSTR_CATEGORY_OTHER,
+
+    /* OP_add */               DR_INSTR_CATEGORY_INT,
+    /* OP_or */                DR_INSTR_CATEGORY_OTHER,
```

Comment:
A bunch of these are not filled in.  That's fine, but it should be clear that this is deliberate, and that this feature is incomplete: the PR description should state that a bunch of opcodes are not yet categorized; the doc comment for the API function should also say something until these are filled in; a TODO comment should go somewhere right here so a read doesn't get really confused thinking you deliberately thought that OR wasn't an integer math operation,

Reply:
add TODO to decode_category body